app.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "Shynet",
+  "description":"Modern, privacy-friendly, and detailed web analytics that works without cookies or JS.",
+    "keywords":[
+      "app.json",
+      "shynet",
+      "heroku",
+      "analytics",
+      "privacy",
+      "friendly"
+  ],
+  "website": "https://github.com/milesmcc/shynet",
+  "repository": "https://github.com/milesmcc/shynet",
+  "logo": "https://github.com/milesmcc/shynet/raw/master/images/slogo.png",
```

Comment:
This 404's. I think the `s` is a typo.

Reply:
It is actually a new smaller image. Curious this scenario where my contribution in fact must point to a broken-URL that still does not exist yet.

### Current:
![](https://raw.githubusercontent.com/thomasgroch/shynet/features/heroku-quick-deploy/images/slogo.png)

`https://github.com/thomasgroch/shynet/raw/features/heroku-quick-deploy/images/slogo.png`

### after merge
![](https://raw.githubusercontent.com/milesmcc/shynet/master/images/slogo.png)

`https://github.com/milesmcc/shynet/raw/master/images/slogo.png`


_Note: the intention is just a quick and dirty crop to look nice on heroku at first. (I'm not realy a designer)_

Reply:
Great, that makes sense.